### DISCUSSION TITLE
Support for Vue 3.4.0

### DISCUSSION THREAD

**Discussion by agracia-foticos** [tailwindlabs/headlessui#2900] [Ideas]
Support for Vue 3.4.0

**Comment by torbentschechne** [comment]
What's not working in Vue 3.4.0?

**Comment by agracia-foticos** [comment]
> What's not working in Vue 3.4.0?

Since many days i updated my project and there were conflicts... Now i have updated and i havent problems. i close the discussion :) Thanks!
